### ISSUE TITLE
USYD Series 5 Exploration

### ISSUE THREAD

**Issue by fantasy121** [OpenSourceMycetoma/series-5-benzimidazoles#2]
![Series 5 Wish list copy](https://user-images.githubusercontent.com/36176821/153989327-69d04450-a5ee-47ac-b8a4-f78dc9b32f5a.png)

@OpenSourceMycetoma/corecontrib I have plans to make more analogues for Series 5. Here are a small library that I want to create for the next few months. This is based off the lead MMV003143 we've got for Series 5 screening, which has excellent in vitro and in vivo efficacy.

**Comment by fantasy121** [comment]
See #1 for other compound suggestions

**Comment by MFernflower** [comment]
@OpenSourceMycetoma/corecontrib  MMV003143 is the internal name of the common veterinary drug Fenbendazole

**Comment by MFernflower** [comment]
@fantasy121 There is also the idea of removing the sulfur atom to remove a possible metabolic liability:

![image](https://user-images.githubusercontent.com/3164942/153989955-127a8b66-1b7b-4859-b7a3-f5a1b3c66900.png)

**Comment by kym834** [comment]
@MFernflower you're reading my mind! I've got this on my list along with one with an amide linker.
I'm going to post in the next week or so my plans for this series.

**Comment by MFernflower** [comment]
https://pubchem.ncbi.nlm.nih.gov/compound/10494218

 @fantasy121  @kym834 

There is the interesting pubchem entry that might be worth synthesizing

**Comment by bebi78** [comment]
@mattodd @MFernflower Here is the research plan of DNDI's benzoxazole open synthesis network project OSN P5. I suppose the group of Louis Maes in Antwerp is testing the compounds against leishmaniasis, but Ben surely knows more about the project. 
As shortly mentioned, I contributed to the DNDi project P1, aminopyrazole compounds. Compounds are very selective. Maybe such N-heterocycles have a fenarimol- or azole-type mode of action. I still have some of the most active compounds of work package H of P1.

[P5.Benzoxazole.pdf](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/files/8120549/P5.Benzoxazole.pdf)
[P1.Aminopyrazole.pdf](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/files/8120572/P1.Aminopyrazole.pdf)

**Comment by kym834** [comment]
@bebi78 @mattodd @MFernflower - P5 would be great to test in regard to Series 5 as it is complementary to those we are looking at synthesising. 

@bendndi is it possible to send some of these benzoxazole compounds to @wwjvdsande for testing?

**Comment by mattodd** [comment]
Have put out a request for sources of compounds, please spread the word or reach out directly
https://twitter.com/MycetOS/status/1496255891695456257

**Comment by bebi78** [comment]
@kym834 @MFernflower @fantasy121 @mattodd Following paper can give a hint at suitable carbamate modifications.

[Carbendazim.derivatives.pdf](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/files/8124072/Carbendazim.derivatives.pdf)

**Comment by MFernflower** [comment]
Thanks @bebi78

**Comment by bebi78** [comment]
@mattodd @MFernflower @wwjvdsande Well, in these black days, especially for those who know people in Ukraine or have served, it is a pleasure to get news from this group while constantly watching the BBC news hoping for peace. In terms of further benzimidazole compound series, as part of a leishmania project, I have a series of pyrimidobenzimidazoles in stock (about 10-12 compounds), which I can share anytime (see attached file for two already published examples). 
Any decision or suggestion about the compounds of my second compound list? As mentioned, the cinnamyls and alkaloids might be especially interesting.

[Pyrimidobenzimidazoles.docx](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/files/8143288/Pyrimidobenzimidazoles.docx)

**Comment by MFernflower** [comment]
@wwjvdsande and @MA-Jjingyi I assume has final say on that matter but yes I do agree about the cinnamyls and alkaloids being possibly useful @bebi78

**Comment by bebi78** [comment]
@MFernflower @wwjvdsande @MA-Jjingyi Thanks. Would be about 16 compounds then. Some of the cinnamyl derivatives are oils or liquids. Is this a problem?

**Comment by MFernflower** [comment]
@bebi78 and @wwjvdsande I know this is a bit out of the box but those cinnamyl compounds almost seem like perfume components??!? 

There is a box of small samples of various perfume components for sale: https://shop.perfumersapprentice.com/p-6632-beginners-aromachemical-kit.aspx - 150 usd without shipping (company is in the US) - most are pure compounds some are mixtures of various things 

I bet they would probably cut down on costs of either the box or the shipping if you explain to them what our research is?

Once again I am not certain of the current state of screening @MA-Jjingyi @wwjvdsande

**Comment by bebi78** [comment]
@MFernflower @mattodd @wwjvdsande @MA-Jjingyi Yes, these cinnamyl compounds have their own and often pleasant smell. Likewise terpenes and vanillin derivatives mentioned in the kit list (citronellol and linalool are also part of my compound list). It is always a pleasure to work with vanillin during Christmas time. Until his retirement in 2014, our academic director held workshops about organic molecules as fragrances for pupils from local grammar schools together with the PhD students of our group and we still have many fragrances in stock. When I was PhD student I took part in the workshops and I had to care for the isolation of clove oil from cloves by steam distillation, and the preparation of a perfume by the combination of six different small molecule esters (actually, the latter was done by the pupils upon guidance). We also had paper models of olfactory receptors bound to Molymod organic model structures of the fragrances to show the pupils teh importance of stereochemistry in this regard. Pure fragrances were also analysed by GC during the workshop. Very often such natural fragrancies have antibiotic and repellent effects protecting the plants, which produce them. Some of them were already confirmed to enhance the antifungal activity fo azoles when combined with them.

**Comment by MA-Jjingyi** [comment]
@bebi78 @MFernflower @wwjvdsande of course. we are interested at those compounds. and we can test the synergy between azoles and these cinnamyl compounds against M.M.  I think it should be operative that the cinnamyl derivatives are oils or liquids. Am I right, Wendy?

**Comment by bebi78** [comment]
@MA-Jjingyi @wwjvdsande I have plenty in stock of those compounds which are oils. So I can send you 1 mL of these oils and you can do the precision weighing for your tests and stock solution preparation then without problem in your lab.

**Comment by wwjvdsande** [comment]
@bebi78. For the oils we tested so far we usually tested based on %v/v. It should therefore be doable to do this in this way. We often make a 50% solution in DMSO first simply to avoid that the oil does not float on top of the culture medium. It worked well this way for tea tree oil.

**Comment by wwjvdsande** [comment]
@OpenSourceMycetoma/corecontrib I looked at the website of DNDi and noted that they have the benzimidazole oxfendazole currently in a phase I study in Tanzania, and flubendazole was tested in the past. Oxfendazole was tested as part of the pandemic response box but unfortunately not active. Flubendazole I did not see in the list yet. 
Carbendazim - active
Fenbendazole - active
Mebendazole- inactive
Oxfendazole - inactive

There are more FDA approved benizimidazole antihelmic agents, such as 
- albendazole,
- ricobenzdazole,
- cyclobendazole
- oxibendazole
- dribendazole
- parbendazole
- bnedazole
- tiabendazole
- triclabendazole
- flubendazole

Would be great if we could test those as well. @dosreic can you help here? Has DNDi some of these compounds so that we can test?

**Comment by bebi78** [comment]
@fantasy121 Do you have albendazole (purchased or selfmade) or other analogs available as mentioned in your first comment here?

**Comment by wwjvdsande** [comment]
Albendazole was modelled with Giardia betatubulin in this paper: https://www.sciencedirect.com/science/article/pii/S2211320721000221
If I take the protein sequence of Giardia and blast it to Madurella the homologue should be KXX77269 (e-value of 0.0 and 81.67% identity): https://www.ncbi.nlm.nih.gov/protein/KXX77269.1? report=genbank&log$=protalign&blast_rank=1&RID=2ZPZ0B4701N
@wosmanitx May be modelling here could also help to understand which molecules would be active and which not? Would be great to see if this is predictive to what we see in vitro. So far all the active ones also were active in vivo.

**Comment by bebi78** [comment]
@wwjvdsande Benzimidazole astemizole was part of the covid response box, quite active against M. m. but less against F. s.

**Comment by memm0740** [comment]
**Marita Emmanuel- New Honours Project focusing on Benzimidazole Synthesis**

Hi,
My name is Marita Emmanuel, and I am doing my honours in synthetic chemistry and chemistry education at the University of Sydney. I have chosen to complete my project by continuing research into the drug discovery for mycetoma and possibly investigate ways to make synthesis safer for high school students. 

So far, I have synthesised one compound from series 2- the aminothiazoles to familiarise myself with the chemistry. However, I intend my main focus to be investigating and synthesising more compounds for series 5- the benzimidazoles. I have planned showing the family of compounds and routes of synthesis I would like to take, shown below. I have made a 2 step synthetic plan to make my lead compound and I have been conducting and evaluating a few different reactions for step 1. 

You can access my GitHub notebook - https://github.com/memm0740/ELN_USYD_Honours/issues. 
I will add more information as I make more progress. My first synthetic goal will be the parent compound and then make modifications. 
Here are a few compounds I would like to attempt to make. I want to keep the core relatively the same as it proved to be the most effective against Madura Mycetomatis shown in the paper- https://doi.org/10.1371/journal.pntd.0010159

![image](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/assets/124969922/ec0a265c-fbc1-4446-b54e-251a9d5ad7e7)

These are the slides I used in Monthly Zoom meeting 2nd May- 
[MycetOS meeting intro.pptx](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/files/11542559/MycetOS.meeting.intro.pptx)
